src/website/apps/api/v2/serializers/team.py

```
@@ -18,18 +18,41 @@ class Meta:
+        data = super().to_representation(instance)
+        if instance.slug:
+            data.pop('id', None)
+        return data
+
     class Meta:
         model = Member
         fields = [
             'id', 'name', 'title', 'about', 'picture_url', 'twitter', 'linked_in',
-            'order', 'created', 'modified'
+            'order', 'created', 'modified', 'public_identifier', 'api_url', 'has_slug'
         ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)